test/Performance/setup_env_windows.ps1

```diff
@@ -0,0 +1,112 @@
+Write-Host "Installing MSODBCSQL..."
+msiexec /quiet /passive /qn /i $tempFolder\msodbcsql.msi IACCEPTMSODBCSQLLICENSETERMS=YES | Out-Null
+Write-Host "Installing 7-Zip..."
+choco install -y 7zip.install
+
+Write-Host "Downloading PHP-SDK..."
+(New-Object System.Net.WebClient).DownloadFile('http://windows.php.net/downloads/php-sdk/php-sdk-binary-tools-20110915.zip', "$tempFolder\binary_tools.zip")
+Write-Host "Downloading PHP-$PHP_VERSION source..."
+IF($PHP_VERSION -eq $PHP70_LATEST_VERSION -Or $PHP_VERSION -eq $PHP71_LATEST_VERSION){
+(New-Object System.Net.WebClient).DownloadFile("http://windows.php.net/downloads/releases/php-$PHP_VERSION-src.zip", "$tempFolder\php-$PHP_VERSION-src.zip")
+}
+ELSE{
+(New-Object System.Net.WebClient).DownloadFile("http://windows.php.net/downloads/releases/archives/php-$PHP_VERSION-src.zip", "$tempFolder\php-$PHP_VERSION-src.zip")
+}
+
+Write-Host "Downloading Dependencies..."
+(New-Object System.Net.WebClient).DownloadFile("http://windows.php.net/downloads/php-sdk/deps-7.$PHP_VERSION_MINOR-vc14-$ARCH.7z", "$tempFolder\deps-7.$PHP_VERSION_MINOR-vc14-$ARCH.7z")
+
+Add-Type -AssemblyName System.IO.Compression.FileSystem
+Remove-Item C:\php-sdk -Recurse -Force -ErrorAction Ignore
+New-Item -ItemType directory -Path C:\php-sdk
+[System.IO.Compression.ZipFile]::ExtractToDirectory("$tempFolder\binary_tools.zip", "C:\php-sdk")
+cd C:\php-sdk\
+bin\phpsdk_buildtree.bat phpdev
+New-Item -ItemType directory -Path .\phpdev\vc14
+Copy-Item .\phpdev\vc9\* phpdev\vc14\ -recurse
+[System.IO.Compression.ZipFile]::ExtractToDirectory("$tempFolder\php-$PHP_VERSION-src.zip", "C:\php-sdk\phpdev\vc14\$ARCH\")
+7z.exe x $tempFolder\deps-7.$PHP_VERSION_MINOR-vc14-$ARCH.7z -oC:\php-sdk\phpdev\vc14\$ARCH\
+
+bin\phpsdk_setvars.bat
+
+cd C:\php-sdk\phpdev\vc14\$ARCH\php-$PHP_VERSION-src
+
+New-Item -ItemType directory -Path .\ext\sqlsrv
+New-Item -ItemType directory -Path .\ext\pdo_sqlsrv
+Copy-Item $DRIVER_SOURCE_PATH\sqlsrv\* .\ext\sqlsrv\ -recurse
+Copy-Item $DRIVER_SOURCE_PATH\shared\ .\ext\sqlsrv\ -recurse
+Copy-Item $DRIVER_SOURCE_PATH\pdo_sqlsrv\* .\ext\pdo_sqlsrv\ -recurse
+Copy-Item $DRIVER_SOURCE_PATH\shared\ .\ext\pdo_sqlsrv\ -recurse
+
+
+$CONFIG_OPTIONS="--enable-cli --enable-cgi --enable-sqlsrv=shared --enable-pdo=shared   --with-pdo-sqlsrv=shared --with-odbcver=0x0380 --enable-mbstring --with-openssl"
+if ($PHP_THREAD -ceq "nts") {
+    $CONFIG_OPTIONS=$CONFIG_OPTIONS + " --disable-zts"
+}
+& $startingDir\compile_php.bat $ARCH $CONFIG_OPTIONS
+
+
+
+
+Copy-Item php.ini-production php.ini
+Add-Content php.ini "extension=C:\php\ext\php_sqlsrv.dll"
+Add-Content php.ini "extension=C:\php\ext\php_pdo_sqlsrv.dll"
+Add-Content php.ini "extension=C:\php\ext\php_openssl.dll"
+Move-Item php.ini C:\Windows -force
+Copy-Item C:\php-sdk\phpdev\vc14\$ARCH\deps\bin\ssleay32.dll C:\Windows -force
+Copy-Item C:\php-sdk\phpdev\vc14\$ARCH\deps\bin\libeay32.dll C:\Windows -force
+
+cd $startingDir

```

Comment:
No, we actually want Composer to be installed in $startingDir. I didnt install composer system-wide. 
You'd use composer.phar in the current directory.